core/src/main/java/org/jboss/set/mjolnir/archive/configuration/ConfigurationProducer.java

```diff
@@ -20,6 +20,7 @@
     private final static String GITHUB_TOKEN_KEY = "github.token";
     private final static String REPOSITORY_ARCHIVE_ROOT_KEY = "application.archive_root";
     private final static String UNSUBSCRIBE_USERS = "application.unsubscribe_users";
+    private final static String CREATE_REMOVALS = "application.create_removals";
```

Comment:
Lukas, can we rename the "create_removals" parameter to "remove_users_without_ldap_account" to make it more specific?